iOSClient/Viewer/NCViewerMedia/NCPlayer/NCPlayerToolBar.swift

```diff
@@ -257,16 +275,24 @@ class NCPlayerToolBar: UIView {
 
     public func show(enableTimerAutoHide: Bool = false) {
 
-        if metadata?.classFile != NCCommunicationCommon.typeClassFile.video.rawValue && metadata?.classFile != NCCommunicationCommon.typeClassFile.audio.rawValue { return }
-        if let metadata = self.metadata, metadata.livePhoto { return }
+        guard let metadata = self.metadata else { return }
+        if ncplayer == nil { return }
+        if metadata.livePhoto { return }
+        if metadata.classFile != NCCommunicationCommon.typeClassFile.video.rawValue && metadata.classFile != NCCommunicationCommon.typeClassFile.audio.rawValue { return }
```

Comment:
Combine into one `guard` for more consistency.

Reply:
> I just have a few questions..
> 
> Also note that I changed the versioning as you asked. Now, to change the version:
> 
> * Open the Nextcloud Xcode project
> * Select **Project** > Nextcloud > Build settings
> * Search or go to "Versioning". There you can change the Build (current project) version as well as the app (marketing) version. Those changes will then automatically be applied to all targets.

Not in File provider extension ...